File: conditionals/time_of_day.c
// Paige Jolley, Time Of Day C
#include <stdio.h>
#include <time.h>
int hour;

int main(void){
    time_t now = time(NULL);
    struct tm * tm_struc = localtime(&now);
int hour = tm_struc->tm_hour;
    printf("%d\n", hour); 

if(hour <= 12){
    printf("Good Morning!\n");
}else if(hour >= 13){
    printf("Good Afternoon!\n");
}else if(hour >= 18){
    printf("Good Evening\n");
}else{
    printf("It is night.\n");
}
    return 0;  
}

File: conditionals/time_of_day.py
# Paige Jolley, Time of day Python
import time

current = time.time()

now = time.ctime(current)

local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour
print(hour)

if hour <= 12: 
    print("Good Morning!\n")
elif hour >= 13:
    print("Good Afternoon!\n")
elif hour >= 18: 
    print("Good Evening!\n")
else: 
    print("It is night.\n")